dataprofiler/tests/profilers/test_int_column_profile.py

```diff
@@ -204,6 +206,56 @@ def batch_variance(mean_a, var_a, count_a, mean_b, var_b, count_b):
         self.assertEqual(variance, num_profiler.variance)
         self.assertEqual(np.sqrt(variance), num_profiler.stddev)
 
+    def test_profiled_skewness(self):
+        data = np.linspace(-5, 5, 11).tolist()
+        df1 = pd.Series(data)
+
+        data = np.linspace(-3, 2, 11).tolist()
+        df2 = pd.Series(data)
+
+        data = np.full((10,), 1)
+        df3 = pd.Series(data)
+
+        num_profiler = IntColumn(df1.name)
+        num_profiler.update(df1.apply(str))
+
+        self.assertAlmostEqual(df1.skew(), num_profiler.skewness)
```

Comment:
instead of utilizing another library, can we specify the actual value here since we know the calculation? Same for the other tests.

Reply:
The list could be more simple if that makes it easier.

Reply:
or at least specify what the value will be as oppose ot calculating.